clients/rust/Cargo.toml

```diff
@@ -8,27 +8,19 @@ readme = "README.md"
 license-file = "../../LICENSE"
 
 [features]
-anchor = ["dep:anchor-lang"]
 test-sbf = []
 serde = ["dep:serde", "dep:serde_with"]
 
 [dependencies]
-anchor-lang = { version = "0.30.0", optional = true }
 borsh = "^0.10"
 num-derive = "^0.3"
 num-traits = "^0.2"
 serde = { version = "^1.0", features = ["derive"], optional = true }
 serde_with = { version = "^3.0", optional = true }
-solana-program = { git = "https://github.com/anza-xyz/agave.git", version = "=2.1.0" }
+solana-program = ">= 1.16, < 3"
```

Comment:
This range might be a bit big, is there a reason for it?

Reply:
No particular reason, do you have a suggestion of which range might be more appropriate?